tests/TestCase.php

```diff
@@ -158,14 +170,21 @@ protected function setUpDatabase($app)
         }
 
         $this->testUser = User::create(['email' => 'test@user.com']);
-        $this->testAdmin = Admin::create(['email' => 'admin@user.com']);
         $this->testUserRole = $app[Role::class]->create(['name' => 'testRole']);
-        $app[Role::class]->create(['name' => 'testRole2']);
-        $this->testAdminRole = $app[Role::class]->create(['name' => 'testAdminRole', 'guard_name' => 'admin']);
         $this->testUserPermission = $app[Permission::class]->create(['name' => 'edit-articles']);
+
+        $this->testAdmin = Admin::create(['email' => 'admin@user.com']);
+        $this->testAdminRole = $app[Role::class]->create(['name' => 'testAdminRole', 'guard_name' => 'admin']);
+        $this->testAdminPermission = $app[Permission::class]->create(['name' => 'admin-permission', 'guard_name' => 'admin']);
+
+        $this->testClient = Client::create(['name' => 'Test', 'redirect' => 'https://google.com', 'personal_access_client' => 0, 'password_client' => 0, 'revoked' => 0]);
```

Comment:
This is the tiniest of suggested changes, but let's use example.com instead of google, to be clear that nothing here is dependent on any google services.